apps/react-18-tests-v9/jest.config.js

```diff
@@ -15,6 +15,10 @@ module.exports = {
       },
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31715&commitId=41a4576e60bcfb5edb58ad77ca8f3c950af8ce5a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.size.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|0| Removed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>